lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -1,112 +1,114 @@
 'use strict';
 
 const BbPromise = require('bluebird');
-const forEach = require('lodash').forEach;
-const merge = require('lodash').merge;
+const _ = require('lodash');
```

Comment:
Is this use of _ a best practice or why are we not giving it a proper name? Its kind of hard to understand what this thing is when the name is _